src/providers/node.rs

```diff
@@ -34,6 +40,37 @@ pub struct PackageJson {
     pub dev_dependencies: Option<HashMap<String, String>>,
 }
 
+#[derive(Debug, Serialize, PartialEq, Deserialize)]
+struct NxJson {
+    #[serde(default)]
+    #[serde(alias = "defaultProject")]
+    default_project: Option<Value>,
+}
+
+#[derive(Debug, Serialize, PartialEq, Deserialize)]
+pub struct Options {
+    #[serde(alias = "outputPath")]
+    output_path: Option<Value>,
+    #[serde(default)]
+    main: Option<Value>,
+}
+
+#[derive(Debug, Serialize, PartialEq, Deserialize)]
+pub struct Build {
+    executor: String,
+    options: Options,
+}
+
+#[derive(Debug, Serialize, PartialEq, Deserialize)]
+pub struct Targets {
+    build: Build,
+}
+
+#[derive(Debug, Serialize, PartialEq, Deserialize)]
+pub struct ProjectJson {
+    targets: Targets,
+}
```

Comment:
Can this be moved into an NX module? It isn't clear that these are just for NX monorepos. I think we should have

```
providers/
  node/
    mod.rs
    nx.rs
```

Can use `git mv providers/node.rs providers/node/mod.rs` to cleanly move it over.

Reply:
Agreed. That git command is neat as well, it worked great once I realised I needed to create the node folder manually 😄.